recipes/pygments/__init__.py

```diff
@@ -0,0 +1,11 @@
+# pure-python package, this can be removed when we'll support any python package
+from toolchain import PythonRecipe
+
+
+class PlyerRecipe(PythonRecipe):
```

Comment:
Shouldn't this be `PygmentsRecipe` instead? :)

Reply:
Yes:-)